evap/static/js/text-answer-warnings.js

```diff
@@ -0,0 +1,45 @@
+function isTextMeaningless(text) {
+    return text.length > 0 && ["", "ka", "na"].includes(text.replace(/\W/g,''));
+}
+
+function doesTextContainTriggerString(text, triggerStrings) {
+    return triggerStrings.some(triggerString => text.includes(triggerString));
+}
+
+function updateTextareaWarning(textarea, textAnswerWarnings) {
+    const text = textarea.val().toLowerCase().replace(/\s{2,}/, " ");
```

Comment:
handling spaces seems not to be consistent. using the trigger string `test_spaces` (`_` standing for a space character) the warning is shown for these texts:
- `this__includes__multiple__test_spaces`
- `this_includes_multiple_test__spaces`

but not for this one:
- `this_includes_multiple__test__spaces`